package.json

```diff
@@ -34,53 +46,30 @@
     "graphql": "^14.5.8",
     "graphql-tag": "^2.10.1",
     "husky": "^3.0.3",
+    "leaflet": "^1.5.1",
+    "lerna": "^3.20.2",
     "lint-staged": "^9.2.1",
     "prettier": "^1.18.2",
+    "prop-types": "^15.7.2",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
     "webpack": "^4.39.2"
   },
-  "main": "core/index.js",
   "scripts": {
-    "preversion": "yarn lint && yarn build",
-    "postversion": "yarn publish --access public --new-version $npm_package_version . && git push --tags && (git diff-index --quiet HEAD || git push) && echo \"Successfully released version $npm_package_version!\"",
-    "postpublish": "yarn deploy",
-    "lint": "yarn eslint --fix --ignore-path './.gitignore' --ext '.js,.json,.md' './'",
-    "lint-staged": "yarn eslint --fix --ignore-path './.gitignore' --ext '.js,.json,.md'",
+    "preversion": "yarn lint",
+    "postversion": "yarn build",
```

Comment:
Shouldn’t build, test, etc. happen _before_ cutting a new version?

Reply:
We wan to build after since i move package.json file into build folder. `lerna publish --contents build` pushes the build folder to npm so it needs to have a package.json with the new version number.